Patches/PlayerControlPatch.cs

```diff
@@ -814,7 +796,7 @@ public static bool Prefix(PlayerControl __instance, [HarmonyArgument(0)] PlayerC
 
                 case CustomRoles.Unlucky:
                     var Ue = IRandom.Instance;
-                    if (Ue.Next(1, 100) <= Options.UnluckyKillSuicideChance.GetInt())
+                    if (Ue.Next(1, 100) <= Unlucky.UnluckyKillSuicideChance.GetInt())
                     {
                         Main.PlayerStates[killer.PlayerId].deathReason = PlayerState.DeathReason.Suicide;
                         killer.RpcMurderPlayerV3(killer);
```

Comment:
not sure what to use for whetever unlucky is dead in prefix.. so for now I don't use the method I made here nor shipstatus ventpatch.

Reply:
You can make
`if (!Unlucky.UnluckCheck) return false;`